internal/gabiactl/gabiactl.go

```
@@ -252,8 +252,8 @@ func validateDesired(d Desired, requireImage bool) error {
 		problems = append(problems, "load_balancer.listeners is required")
 	}
 	for _, l := range d.LoadBalancer.Listeners {
-		if !((l.Port == 80 && l.Protocol == "HTTP") || (l.Port == 443 && l.Protocol == "HTTPS")) || l.TargetPort < 1 || l.TargetPort > 65535 {
-			problems = append(problems, "load balancer listeners allow only HTTP/80 or HTTPS/443 with a valid target_port")
+		if !((l.Port == 80 && l.Protocol == "HTTP") || (l.Port == 443 && (l.Protocol == "HTTPS" || l.Protocol == "TCP"))) || l.TargetPort < 1 || l.TargetPort > 65535 {
```

golangci-lint (2.12.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)